File: transformer/db/models/atm.py
from peewee import *
from .office import Position
from .base_model import BaseModel

import uuid


class Atm(BaseModel):
  id = UUIDField(primary_key=True, default=uuid.uuid4)
  address = CharField(max_length=255)
  all_day = BooleanField()
  position = ForeignKeyField(Position, related_name='position')

File: server/internal/entity/atm.go
package entity

import "github.com/gofrs/uuid/v5"

type ATM struct {
	ID       uuid.UUID `json:"id"`
	Position *Position `json:"position"`
	Address  string    `json:"address"`
	AllDay   bool      `json:"all_day"`
	Services []Service `json:"services"`
}
